main/php5/php-fpm.initd

```diff
@@ -8,14 +8,13 @@
 
 : ${name:="PHP FastCGI Process Manager"}
 
-command="/usr/bin/php-fpm"
+command="/usr/sbin/php-fpm5"
```

Comment:
It was a mistake when original php5 apkbuild was created. php-fpm binary natively lives in /usr/sbin

I made a compatibility link to /usr/bin